app/src/main/java/edu/grinnell/kdic/MainActivity.java

```diff
@@ -374,13 +380,13 @@ public void showVisualizeFragment() {
             mPlaybackToolbar.setNavigationIcon(R.drawable.ic_keyboard_arrow_down_white_24dp);
 
             // move the play button to the middle
-            final float shiftAmnt = (mPlaybackToolbar.getWidth() - playPauseButton.getWidth()) / 2;
+            final float shiftAmnt = (mPlaybackToolbar.getWidth() - mPlayPauseButton.getWidth()) / 2;
             TranslateAnimation animation = new TranslateAnimation(shiftAmnt, 0, 0, 0);
```

Comment:
0's here do not make sense to anyone reading the code. You might want to extract these as constants and give them descriptive names.